docs/content/engines/flink.md

```diff
@@ -210,3 +210,19 @@ All Flink data types are supported, except that
 
 * `MULTISET` is not supported.
 * `MAP` is not supported as primary keys.
+
+## Use Flink Managed Memory
+In order to improve the buffer pool for writers in Paimon tasks.Paimon tasks can create memory pools based on Executor Memory which will be managed by Executor, such as Managed Memory in Flink TaskManager. It will improve the stability and performance of sinks by managing writer buffers for multiple tasks through Executor.
```

Comment:
```suggestion
Paimon tasks can create memory pools based on executor memory which will be managed by Flink executor, such as managed memory in Flink task manager. It will improve the stability and performance of sinks by managing writer buffers for multiple tasks through executor.
```